preview.netcdf/requirements.txt

```diff
@@ -0,0 +1,7 @@
+gsconfig-py3==1.0.8
+pika>=1.0.0
+requests>=2.10.0
+wheel>=0.24.0
+pyclowder
```

Comment:
You might want to specify pyclowder to be >=3.0.7 Just to be safe?

Reply:
fixed clowder version